app/Http/Middleware/AssetCountForSidebar.php

```diff
@@ -59,6 +67,44 @@ public function handle($request, Closure $next)
             \Log::debug($e);
         }
 
+        try {
+            $settings = Setting::getSettings();
+            view()->share('settings', $settings);
+        } catch (\Exception $e) {
+            \Log::debug($e);
+        }
+
+        try {
+            $total_due_for_audit = Asset::DueForAudit($settings)->count();
+            view()->share('total_due_for_audit', $total_due_for_audit);
+        } catch (\Exception $e) {
+            \Log::debug($e);
+        }
+
+        try {
+            $total_overdue_for_audit = Asset::OverdueForAudit($settings)->count();
```

Comment:
Double-check but `$settings` shouldn't be needed here:

```suggestion
            $total_overdue_for_audit = Asset::OverdueForAudit()->count();
```